src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOTalon.kt

```diff
@@ -210,13 +210,12 @@ class SwerveModuleIOTalon(
     updateSignals()
 
     inputs.driveAppliedVoltage = (driveFalcon.get() * RobotController.getBatteryVoltage()).volts
-    inputs.steeringAppliedVoltage =
-      (steeringFalcon.get() * RobotController.getBatteryVoltage()).volts
+    inputs.steerAppliedVoltage = (steeringFalcon.get() * RobotController.getBatteryVoltage()).volts
```

Comment:
has to be a better way, i don't trust this method of getting applied voltage

Reply:
changed is it good

Reply:
😎